tests/unit/amo/reducers/test_users.js

```diff
@@ -445,4 +450,172 @@ describe(__filename, () => {
       expect(isDeveloper(user)).toEqual(false);
     });
   });
+
+  describe('unsubscribeNotification', () => {
+    it('sets `isUnsubscribedFor` to `false`', () => {
+      const hash = 'some-hash';
+      const notification = 'new_review';
+      const token = 'some-token';
+
+      const state = reducer(
+        undefined,
+        unsubscribeNotification({
+          errorHandlerId: 'error-handler-id',
+          hash,
+          notification,
+          token,
+        }),
+      );
+
+      expect(isUnsubscribedFor(state, hash, notification, token)).toEqual(
+        false,
+      );
+    });
+
+    it('does not change the value of another notification', () => {
+      const hash = 'some-hash';
+      const token = 'some-token';
+      const notification1 = 'notification1';
```

Comment:
To improve readability of the test, you might consider renaming `notification1` to `updatedNotification` and `notification2` to `pendingNotification`.